src/data/faq.json

```diff
@@ -282,7 +282,7 @@
                                 "anchor": "rat_multiple_rat",
                                 "active": false,
                                 "textblock": [
-                                    "No, only one rapid test can be registered in the app at a time. A negative rapid test is displayed in the app for 48 hours. After this time you can register a new test. You also have the option to remove the negative rapid test and result, and then register a new rapid test. Positive rapid tests are no longer displayed in the app once they have been shared.",
+                                    "Yes, since version 2.21 it is possible to manage quick tests of family members in the Corona-Warn-App. For each person continues to apply: A negative rapid test is displayed in the app for 48 hours. After this time you can register a new test. You also have the option to remove the negative rapid test and result, and then register a new rapid test. Positive rapid tests are no longer displayed in the app once they have been shared.",

```

Comment:
"For each person continues to apply:" =>
"For each person the following continues to apply:"

Reply:
Changed in latest commit, thanks for the suggestion